realm/realm-annotations-processor/src/test/resources/some/test/Counter_Ignored.java

```diff
@@ -0,0 +1,16 @@
+package some.test;
+
+import io.realm.MutableRealmInteger;
+import io.realm.RealmObject;
+import io.realm.annotations.Ignore;
+
+public class Counter_Ignored extends RealmObject {
```

Comment:
For those kind of test, you can actually use `TestRealmObjectFileObject` which doesn't need to create a model classes anymore and make the tests much easier. See `compileInvalidRequiredTypes` for example.